ecf/scripts/cam/plots/jevs_cam_precip_plots.ecf

```diff
@@ -60,32 +59,29 @@ module load zlib/${zlib_ver}
 module load jasper/${jasper_ver}
 module load udunits/${udunits_ver}
 module load gsl/${gsl_ver}
-module load hdf5/${hdf5_ver}
-module load python/${python_ver}
 module load netcdf/${netcdf_ver}
 module load nco/${nco_ver}
 module load prod_util/${prod_util_ver}
 module load prod_envir/${prod_envir_ver}
 module load cdo/${cdo_ver}
-module use /apps/ops/prod/libs/modulefiles/compiler/intel/${intel_ver}
 module load grib_util/${grib_util_ver}
 module load wgrib2/${wgrib2_ver}
 module load proj/${proj_ver}
 module load geos/${geos_ver}
 module load met/${met_ver}
 module load metplus/${metplus_ver}
-export MET_PLUS_PATH="/apps/ops/prod/libs/intel/${intel_ver}/metplus/${metplus_ver}"
-export MET_PATH="/apps/ops/prod/libs/intel/${intel_ver}/met/${met_ver}"
+export MET_PLUS_PATH="/apps/ops/para/libs/intel/${intel_ver}/metplus/${metplus_ver}"
+export MET_PATH="/apps/ops/para/libs/intel/${intel_ver}/met/${met_ver}"
 export MET_CONFIG="${MET_PLUS_PATH}/parm/met_config"
 export PYTHONPATH=$HOMEevs/ush/$COMPONENT:$PYTHONPATH
 
 # Developer Settings
 export DATA=/lfs/h2/emc/stmp/$USER/evs_test/$envir/tmp
-export COMINccpa=/lfs/h2/emc/ptmp/${USER}/EVS_out/com/$NET/$evs_ver/prep/$COMPONENT/$RUN
-export COMINmrms=/lfs/h2/emc/ptmp/${USER}/EVS_out/com/$NET/$evs_ver/prep/$COMPONENT/$RUN
-export COMINspcotlk=/lfs/h2/emc/ptmp/${USER}/EVS_out/com/$NET/$evs_ver/prep/$COMPONENT/$RUN
-export COMIN=/lfs/h2/emc/ptmp/${USER}/EVS_out/com/$NET/$evs_ver/stats/$COMPONENT/$RUN
-export COMOUT=/lfs/h2/emc/ptmp/${USER}/EVS_out/com/$NET/$evs_ver/$STEP/$COMPONENT/$RUN
+export COMINccpa=/lfs/h2/emc/ptmp/${USER}/EVS_out/com/$NET/$evs_ver/prep/$COMPONENT
+export COMINmrms=/lfs/h2/emc/ptmp/${USER}/EVS_out/com/$NET/$evs_ver/prep/$COMPONENT
+export COMINspcotlk=/lfs/h2/emc/ptmp/${USER}/EVS_out/com/$NET/$evs_ver/prep/$COMPONENT
+export COMIN=/lfs/h2/emc/vpppg/noscrub/${USER}/$NET/$evs_ver/stats/$COMPONENT
+export COMOUT=/lfs/h2/emc/vpppg/noscrub/${USER}/$NET/$evs_ver/$STEP/$COMPONENT
```

Comment:
Remove `$STEP/$COMPONENT` from line 84 b/c $COMOUT is used in the definition for $COMOUTplots in `JEVS_CAM_PLOTS`:
`export COMOUTplots=$COMOUT/$STEP/$COMPONENT/$RUN.${VDATE}`